lib/pbench/cli/server/tree_manage.py

```diff
@@ -88,18 +109,33 @@ def tree_manage(
         search: Discover cache with a full disk search
     """
     logger = None
+
+    global detailer, verifier, watcher
+    detailer = Detail(detail, False)
+    verifier = Verify(verify)
+    watcher = Watch(progress)
+
     try:
         config = config_setup(context)
         logger = get_pbench_logger("pbench-tree-manager", config)
         cache_m = CacheManager(config, logger)
         if display:
+            verifier.status("starting discovery")
+            watcher.update("discovering cache")
             cache_m.full_discovery(search=search)
+            verifier.status("finished discovery")
+            watcher.update("building report")
             print_tree(cache_m)
             rv = 0
         if reclaim_percent or reclaim_size:
+            verifier.status("starting reclaim")
+            watcher.update("reclaiming")
             target_size = humanfriendly.parse_size(reclaim_size) if reclaim_size else 0
             target_pct = reclaim_percent if reclaim_percent else 20.0
```

Comment:
As long as we've got this module open....

I'm nearly certain that we don't need the ternary for `target_pct`, because I believe that Click will set `reclaim_percent` to `20.0` to per line 66 if the user doesn't set it (so we'll never use the `else` value).

I think something similar could be arranged for `target_size`.

Reply:
You might think so, wouldn't you? But at least in my initial experiments it didn't work that way. Maybe I'll experiment some more. I'd still love to default `--config` to `/opt/pbench-server/lib/config/pbench-server.cfg` in the absence of an environment variable, though that might require a custom option callback. Perhaps I'll waste some time "learning" today in lieu of the Day of Learning we worked through yesterday. 😁

Reply:
I did some experimenting and I think I've achieved what I'd originally wanted but didn't quite have. (Although, in retrospect, I'm not entirely sure why I really wanted the % to default to 20, I guess I'm keeping it.)

My intent was that we *don't* reclaim cache if you don't specify either option. That isn't actually working right now since `pbench-tree-manage` defaults reclaim percent as 20.0. But I wanted `pbench-tree-manage --percent-reclaim` to default to 20%; and that's not working, either.

So I've revised the option definition so that it works as I think I'd expected, and maybe more or less like what I want ... or at least something plausibly useful.

Anyway, ultimately the point of the ternaries are that either may be `None` but I don't want to pass that on to the reclaimer. However, I think that if `reclaim_percent` is `None` I really want to contribute `0.0` rather than `20.0` because that means the user has specified `--reclaim-size <n>` and I want to reclaim (only) `<n>` even if it's less than 20%.

```
/opt/pbench-server/bin/pbench-tree-manage --reclaim-percent 60 --reclaim-size 1Mb --verify
(14:28:33) starting to reclaim 60.0% or 1000000
/opt/pbench-server/bin/pbench-tree-manage --reclaim-percent --reclaim-size 1Mb --verify
(14:28:43) starting to reclaim 20.0% or 1000000
/opt/pbench-server/bin/pbench-tree-manage --reclaim-percent --verify
(14:34:46) starting to reclaim 20.0% or 0
/opt/pbench-server/bin/pbench-tree-manage --reclaim-size 1Mb --verify
(14:33:11) starting to reclaim 0.0% or 1000000
/opt/pbench-server/bin/pbench-tree-manage --verify
# (Nada zip zilch)
```